charts/apisix/README.md

```diff
@@ -0,0 +1,122 @@
+## Apache APISIX for Kubernetes
+
+Apache APISIX is a dynamic, real-time, high-performance API gateway.
+
+APISIX provides rich traffic management features such as load balancing, dynamic upstream, canary release, circuit breaking, authentication, observability, and more.
+
+You can use Apache APISIX to handle traditional north-south traffic, as well as east-west traffic between services. It can also be used as a [k8s ingress controller](https://github.com/apache/apisix-ingress-controller/).
+
+This chart bootstraps all the components needed to run Apache APISIX on a Kubernetes Cluster using [Helm](https://helm.sh).
+
+
+## TL;DR
+
+```sh
+➜ helm repo add apisix https://charts.apiseven.com
+➜ helm repo update
```

Comment:
What about adding a new step like:

```sh
helm search repo apisix
NAME                                    CHART VERSION   APP VERSION     DESCRIPTION
apisix/apisix                           0.3.2           2.5.0           A Helm chart for Apache APISIX
apisix/apisix-dashboard                 0.1.2           2.5.0           A Helm chart for Apache APISIX Dashboard
apisix/apisix-ingress-controller        0.4.0           0.5.0           Apache APISIX Ingress Controller for Kubernetes
```

So that people can get the details about this repo.

Reply:
In fact, I only want to expose the apisix/apisix repo to users. Other repositories are not for users to use directly.

Reply:
OK.